src/components/SupportRequest.js

```diff
@@ -233,7 +233,7 @@ const SupportRequest = () => {
 const SupportRequestWrapper = () => {
   const isActive = useStore(contactUsActive)
 
-  return isActive && h(SupportRequest)
+  return isActive ? h(SupportRequest) : null
```

Comment:
Components are intended to return a ReactElement or null.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e6c770ac6c25dcc7590f22e50a3f7dc582887da3/types/react/index.d.ts#L520-L526